docs/source/quantization-support.rst

```diff
@@ -1,12 +1,12 @@
 Quantization API Reference
 -------------------------------
 
-torch.quantization
-~~~~~~~~~~~~~~~~~~
+torch.ao.quantization
```

Comment:
nit: we haven't communicated about torch.ao migration yet, it would be good for that change to be in the same PR as the PR that adds deprecation warnings.  It might be confusing to add it to the docs now before we have a public facing explanation of the change.

Reply:
oh ok, sounds good

Reply:
some of the modules are only available in torch.ao, e.g. torch.ao.quantization.qconfig_mapping, is it OK to add them now?